go/libraries/doltcore/schema/typeinfo/time.go

```diff
@@ -63,10 +63,18 @@ func (ti *timeType) ConvertValueToNomsValue(ctx context.Context, vrw types.Value
 	if v == nil {
 		return types.NullValue, nil
 	}
-	val, err := ti.sqlTimeType.Marshal(v)
-	if err != nil {
-		return nil, err
+	var val int64
+	switch v.(type) {
+	case int64:
+		val = v.(int64)
```

Comment:
This line of code doesn't make sense. Even if v is already an int64 it should still be marshalled appropriately.